python/lsst/obs/base/ingest_tests.py

```diff
@@ -238,23 +239,52 @@ def testFailOnConflict(self):
             self._ingestRaws(transfer="symlink")
 
     def testWriteCuratedCalibrations(self):
-        """Test that we can ingest the curated calibrations"""
+        """Test that we can ingest the curated calibrations, and read them
+        with `loadCamera` both before and after.
+        """
         if self.curatedCalibrationDatasetTypes is None:
             raise unittest.SkipTest("Class requests disabling of writeCuratedCalibrations test")
 
+        butler = Butler(self.root, writeable=False)
+        collection = self.instrumentClass.makeCalibrationCollectionName()
+
+        # Trying to load a camera with a data ID not known to the registry
+        # is an error, because we can't get any temporal information.
+        with self.assertRaises(LookupError):
+            lsst.obs.base.loadCamera(butler, self.dataIds[0], collections=collection)
+
+        # Ingest raws in order to get some exposure records.
+        self._ingestRaws(transfer="link")
```

Comment:
Why is this not "auto"?

Reply:
Was just copying from tests above.  Will change.